### PR TITLE
Add media query min-width 768px to index.css

### PR DESCRIPTION
**Pull request by EmekaEmmanuel**
A simple portfolio website.
 Below are the sections contained in this project;
- Toolbar
- Headline section (right after the header).
- Works
- About myself
- Contact me
The layout was done using css flexbox and css grid, button interactions implemented.
Use media queries to make sure that website displays correctly on desktop screens. (Desktop: 768px or wider).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,55 +18,78 @@
...
+                <li class="list_style"> <a class="textdecoration" href="#">About</a></li>
+                <li class="list_style"><a class="textdecoration" href="#">Contact</a></li>
+            </ul>
+
+
+        </nav>
+
     </header>
 
     <section class="section1">
```
Comment (Gopxfs) [inline index.html]: https://github.com/EmekaEmmanuel/portfolio_mobile_day3/blob/d910d203c2003cf8d6a80ff48513ee9ef14b030e/index.css#L32-L39

Please update your `.section1` to have the corresponding background-image in the desktop version 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| image | image  |

> Notice how your background-image is larger, looks like it is zoomed in.

File: index.css [hunk 2]
```diff
@@ -388,3 +403,213 @@ label {
...
+
+  .section1 p {
+    color: #344563;
+    font-size: 13px;
+    line-height: 28px;
+    width: 580px;
+    font-weight: 400;
+    align-items: left;
+    padding: 50px 0 0 0;
+  }
```
Comment (Gopxfs) [inline index.css]: Hello @EmekaEmmanuel, I've noticed that you are using the wrong typography for a lot of elements in your website. Please notice that the desktop version requires a different scale for your elements 😉. But don't worry, you can always check for the whole typography in Figma. To do so, just hover over any element and click on it, a window will be displayed with the correct properties values:
image

I've checked the following elements, please apply the typography display in Figma to them:
- .section paragraph:
image
- .articlediv paragraph:
image
- .about_self_section paragraph:
image
- languages text:
image
- .contact-form paragraph:
image

If you have any doubts please tag me 😄.

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.

File: index.html [hunk 3]
```diff
@@ -94,17 +117,46 @@ <h3><a class="textdecoration font1" href="#">Tonic</a></h3>
...
+                <ul class="gridbox1">
+
+                    <li class="background1"><a class="textdecoration font4" href="#">html</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">Ruby on rails</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">css</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">javaScript</a></li>
+
+                </ul>
+
+                <button type="submit"><a class="font5 textdecoration" href="#">See Project</a></button>
```
Comment (Gopxfs) [inline index.html:146]: https://github.com/EmekaEmmanuel/portfolio_mobile_day3/blob/d910d203c2003cf8d6a80ff48513ee9ef14b030e/index.css#L230-L239

Please notice that the Figma's button doesn't contain a border 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

To remove it, you could do:
```css
.button {
    border: 0;
}
```

File: index.css [hunk 4]
```diff
@@ -388,3 +403,213 @@ label {
...
+  .nav2 a {
+    font-size: 15px;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  .section1 {
+    display: flex;
+    justify-content: center;
+  }
```
Comment (Gopxfs) [inline index.css]: Please notice that your section isn't filling the whole screen:
image

Kindly adjust its width to make your project stretch to any screen size 😉.

Note: just like you did in the contact section 😄:
image

File: index.css [hunk 5]
```diff
@@ -388,3 +403,213 @@ label {
...
+
+  .works_section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 90%;
+    margin: auto;
+    padding: 20px 0 30px 0;
+  }
```
Comment (Gopxfs) [inline index.css]: Please follow the works_section article separation as displayed in Figma 😃:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  | 

Notice how the original background color of the body is different, and each article is a different card with rounded borders 😉.

File: index.css [hunk 6]
```diff
@@ -388,3 +403,213 @@ label {
...
+
+  .works_section .article1,
+  .works_section .article2,
+  .works_section .article3,
+  .works_section .article4 {
+    display: flex;
+    justify-content: center;
+    gap: 15px;
+    margin: 20px 0;
+  }
```
Comment (Gopxfs) [inline index.css]: Please adjust the width of your work_section articles 😀:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

Notice that in the original the card occupies almost 80% of the screen and the spacing of the elements inside it is larger 😉.

File: index.css [hunk 7]
```diff
@@ -388,3 +403,213 @@ label {
...
+    font-weight: 500;
+  }
+
+  .about_self_section {
+    display: flex;
+    justify-content: center;
+    width: 90%;
+    margin: auto;
+    padding: 70px 0 100px 0;
+  }
```
Comment (Gopxfs) [inline index.css]: Please adjust the width of your about myself section😀:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

Notice that in the original the card occupies the whole screen and the spacing of the elements inside it is larger 😉.

File: index.css [hunk 8]
```diff
@@ -388,3 +403,213 @@ label {
...
+    font-size: 20px;
+    align-items: left;
+    font-weight: 500;
+    line-height: 24px;
+  }
+
+  .about_section_flex3 {
+    display: flex;
+    gap: 10px;
+  }
```
Comment (Gopxfs) [inline index.css]: Please adjust the text color of the elements to be the same as the one in Figma:
image

Also, please adjust the width of the language boxes. Notice that all have the same width 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| image| image  |

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @EmekaEmmanuel 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @EmekaEmmanuel , 

Good work implementing the changes requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**